Gemfile

```diff
@@ -3,6 +3,7 @@ source 'https://rubygems.org'
 gem "manageiq-messaging", '~> 0.1.2'
 
 gem "inventory_refresh",          :git => "https://github.com/ManageIQ/inventory_refresh",          :branch => "master"
+gem "manageiq-loggers", "~> 0.1.0"
```

Comment:
Minor, but should this be up with the other released gem(s)?

Reply:
:+1: I was ordering alphabetically but missed that one up there

Reply:
Done